packages/components/src/breadcrumbs/types.ts

```diff
@@ -0,0 +1,36 @@
+export interface BreadcrumbItemProps {
+	/**
+	 * The label text for the breadcrumb item.
+	 */
+	label: string;
+	/**
+	 * The URL that the breadcrumb item should link to.
+	 */
+	href?: string;
+	/**
+	 * An optional callback to handle clicking on this breadcrumb item.
+	 */
+	onClick?: React.MouseEventHandler;
+}
+
+export interface BreadcrumbProps {
+	/**
+	 * An array of items to display in the breadcrumb trail.
+	 * The last item is considered the current page.
+	 */
+	items: BreadcrumbItemProps[];
+	/**
+	 * A boolean to show/hide the current page in the trail.
+	 * Note that when `false` the current page is only visually hidden.
+	 * @default true
+	 */
+	showCurrentPage?: boolean;
```

Comment:
if the items are called "items", should this one be called `showActiveItem` or `showCurrentItem`?